### PR TITLE
Don't crash on shlex.split error

### LINKED ISSUE #107: Apostrophes in prompt mode cause crashes
```
topydo> add read matt's draft
Traceback (most recent call last):
  File "/home/andreas/pool/bin/topydo", line 11, in <module>
    sys.exit(main())
  File "/home/andreas/pool/local/lib/python2.7/site-packages/topydo/cli/UILoader.py", line 45, in main
    PromptApplication().run()
  File "/home/andreas/pool/local/lib/python2.7/site-packages/topydo/cli/Prompt.py", line 103, in run
    user_input = shlex.split(user_input)
  File "/home/andreas/pool/local/lib/python2.7/site-packages/ushlex.py", line 298, in split
    else:       return list(lex)
  File "/home/andreas/pool/local/lib/python2.7/site-packages/ushlex.py", line 283, in next
    token = self.get_token()
  File "/home/andreas/pool/local/lib/python2.7/site-packages/ushlex.py", line 99, in get_token
    raw = self.read_token()
  File "/home/andreas/pool/local/lib/python2.7/site-packages/ushlex.py", line 185, in read_token
    raise ValueError, "No closing quotation"
ValueError: No closing quotation
```

### PR DESCRIPTION
**Pull request by mruwek**
Show some meaningful error message instead.

Fixes #107

### CODE REVIEW

**Comment by bram85** [comment]
I think a similar fix is also needed in `Config.aliases()`

**Comment by mruwek** [comment]
> I think a similar fix is also needed in Config.aliases()

Ok, I'll look into it and amend the commit above.

**Comment by mruwek** [comment]
> Would it be possible to return None instead of 'error'

It would cause another problem, as we use `None` whenever we want to show `usage()`.

> What if I'd define an alias called error?

Valid concern. Maybe it should return `False` or `0` (int type)?

**Comment by bram85** [comment]
> Valid concern. Maybe it should return `False` or `0` (int type)?

No, raising an exception would be more appropriate, which can also replace returning `None`.

File: topydo/Commands.py [hunk 1]
```diff
@@ -90,7 +90,11 @@ def resolve_alias(p_alias, p_args):
         If alias resolves to non-existent command, main help message is
         returned.
         """
-        real_subcommand, alias_args = alias_map[p_alias]
+        try:
+            real_subcommand, alias_args = alias_map[p_alias]
+        except ValueError:
+            raise ConfigError(alias_map[p_alias])
```
Comment (bram85) [inline topydo/Commands.py]: I think the exception should be linked with the `as` and `from` syntax.

**Comment by bram85** [comment]
Change looks good bar a minor remark. We must not forget to catch `ConfigError` in the column-ui branch.

File: topydo/Commands.py (hunk 1 above)
Comment (mruwek) [inline topydo/Commands.py]: Done in b2f34ad.

**Comment by mruwek** [comment]
> We must not forget to catch ConfigError in the column-ui branch.

I would suggest: `topydo add (B) Catch unclosed quote ConfigError in column-ui after:nwh`

**Pull request merged**